server/src/main/java/org/opensearch/cluster/service/Throttler.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.cluster.service;
+
+import org.opensearch.common.AdjustableSemaphore;
+
+import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+/**
+ * Base class for Throttling logic.
+ * It provides throttling functionality over multiple keys.
+ * It provides the functionality of enable/disable throttling using enableThrottling variable.
```

Comment:
This comment seems to be out of context: there is no `enableThrottling` variable nor enable / disable functionality.

Reply:
I missed to remove from here. Thanks for catching it. Removed it.